src/utils/CrossChainBridge.ts

```diff
@@ -160,11 +180,14 @@ export const handleFederatorVoted = (event: VotedEvent, transaction: Transaction
   // TODO: if token is native to RSK, then symbol should be from token entity and not side token
   crossTransfer.symbol = event.params.symbol
   crossTransfer.sender = event.params.sender
-  crossTransfer.votes = crossTransfer.votes + 1
+  if (isSignatureFederation(event.address.toHex().toLowerCase())) {
+    crossTransfer.votes = crossTransfer.votes = 3
```

Comment:
I don't understand this line

Reply:
can you be more specific?
you don't understand why votes should equal 3?